### ISSUE TITLE
Cannot use GPU on CST in Poly-Grames node006/node007

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/neuropoly-internal-docs#61]
@4rnaudB in addition to documenting your debugging, can you pls also describe a way to test a simulation (eg: open a 'dummy' project in CST, click on solver, etc.)
--> [Presentation on how to set up the GPU acceleration ](https://docs.google.com/presentation/d/1owpKA0jjhQlay2ysMWN0vxNOyDZbTJs7Tl9qFlf_ugw/edit#slide=id.g13ee4557173_0_22)

Relevant documentation: https://updates.cst.com/downloads/GPU_Computing_Guide_2022.pdf

**Comment by jcohenadad** [comment]
> administrator privilege would be required to set environment variables

so we need to loop in JS-- i'll take care of it

**Comment by jcohenadad** [comment]
testing on node006, I tried launching `HWAccDiagnostics`, but got this:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/2482071/178032353-afda2c18-67c9-4999-81a5-f808e430b21f.png">
 Likely related to https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/61#issuecomment-1179118610

**Comment by 4rnaudB** [comment]
trying it right now

**Comment by JustinDeM** [comment]
We get this error message :
<img width="559" alt="Screen Shot 2022-07-08 at 12 42 24" src="https://user-images.githubusercontent.com/105290244/178034880-32cfc585-1405-4849-aa93-fa15a497987f.png">

**Comment by 4rnaudB** [comment]
Seems we don't have access to C:\ we can't run it. There is another test you can do if you have access to see if the GPU is recognized.
![image](https://user-images.githubusercontent.com/55560824/178035276-43c6bab2-d0a7-41e3-86e0-45f63950d28a.png)

**Comment by 4rnaudB** [comment]
Also, we got an error this morning with node006 saying no GPU was recognized but we don't get the same error with node007 when running the simulation.

**Comment by JustinDeM** [comment]
When we run the `Check GPU Computing` macro on Node007, we get this : 
<img width="731" alt="Screen Shot 2022-07-08 at 12 53 15" src="https://user-images.githubusercontent.com/105290244/178037031-ae11498f-96ff-4fb9-a069-0ec1e9eb4b86.png">

(The Tesla-V100s is a Volta series).

**Comment by jcohenadad** [comment]
> There is another test you can do if you have access to see if the GPU is recognized.

I don't have access-- hopefully a Poly admin will help us ASAP

**Comment by jcohenadad** [comment]
> When we run the Check GPU Computing macro on Node007, we get this :

Ah! That's interesting, when I ran it on node006 I got the error message reported in https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/61#issuecomment-1179169635

**Comment by JustinDeM** [comment]
Error Correction Code. We are trying to change it but the Nvidia Control Panel won't open.

**Comment by jcohenadad** [comment]
> It could be related to the fact that someone else is using a lot of the ressources of Node007 right now :

Good point-- we should ask the GRAMES admin if they have a way to allocate ressource via a priority list, as done in Compute Canada with SLURM.

**Comment by jcohenadad** [comment]
Answer from JS:
> Ce serveur vient d’être monté il n’y a pas ci longtemps. J’ai vu le message sur GitHub, je vais ajouter la variable un peu plus tard… Pour le moment seulement ADS été utilisé sur ces serveurs

**Comment by 4rnaudB** [comment]
> @4rnaudB in addition to documenting your debugging, can you pls also describe a way to test a simulation (eg: open a 'dummy' project in CST, click on solver, etc.)
> 
> Relevant documentation: https://updates.cst.com/downloads/GPU_Computing_Guide_2022.pdf

Should this be an issue or a powerpoint? (I'm planning to put images for a step by step process)

**Comment by jcohenadad** [comment]
> Should this be an issue or a powerpoint? (I'm planning to put images for a step by step process)

hum, good question. If you prefer creating a PPTX, then I suggest you do it with Gslide, put the document [here](https://drive.google.com/drive/folders/1YZ0Dhw7gdCrqOAKLuPh1Pz-kYQPtljJm) and link it in this issue?

**Comment by 4rnaudB** [comment]
Sounds good

**Comment by 4rnaudB** [comment]
The problem is that the link to the google slide would be at the end of the issue so it might not be very intuitive/practical to add it in this issue.

**Comment by jcohenadad** [comment]
> The problem is that the link to the google slide would be at the end of the issue so it might not be very intuitive/practical to add it in this issue.

so how about adding it at the top? https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/61#issue-1299137510

**Comment by 4rnaudB** [comment]
Oh nice, didn't know I could edit your comment.
